### ISSUE TITLE
LRP SOFTMAX

### ISSUE THREAD

**Issue by andreapi87** [PrimozGodec/keras-explain#7]
If I ask for an explanation on a model with softmax as last layer i get:
LayerNotImplementedException
I think it can be easily removed simply not considering the last softmax layer
